extensions/src/main/java/org/apache/falcon/extensions/jdbc/ExtensionMetaStore.java

```diff
@@ -154,7 +157,7 @@ public void storeExtensionJob(String jobName, String extensionName, List<String>
         EntityManager entityManager = getEntityManager();
         try {
             beginTransaction(entityManager);
-            entityManager.persist(extensionJobsBean);
+            entityManager.merge(extensionJobsBean);
```

Comment:
Why has been changed to merge?As I understand, merge allows for updates. In this particular API, we only want to allow creation. Isn't it?